aws-s3-transfer-manager/src/middleware/hedge.rs

```diff
@@ -0,0 +1,84 @@
+/*
+ * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+use std::time::Duration;
+use tower::{
+    hedge::{Hedge, Policy},
+    layer::layer_fn,
+    BoxError, Layer, Service,
+};
+
+/// S3 recommends retrying the slowest 5% of the requests.
+const LATENCY_PERCENTILE: f32 = 95.0;
+/// This value was chosen randomly and empirically tested.
```

Comment:
utterly trivial: "empirically tested" sounds like there is actually some science behind it
```suggestion
/// This value was chosen randomly, but seems to work.
```